packages/core/src/__tests__/productivity-checker.test.ts

```
@@ -0,0 +1,380 @@
+ */
+
+import { describe, it, expect, vi, beforeEach, afterEach } from "vitest";
+import type { ProductivityDeps } from "../productivity-checker.js";
+import {
+  checkMergedPRCleanup,
+  checkStallDetection,
+  checkContextExhaustion,
+  runProductivityChecks,
+  STALL_THRESHOLD_MS,
+  NUDGE_COOLDOWN_MS,
+} from "../productivity-checker.js";
```

ESLint

[error] '../productivity-checker.js' import is duplicated.

(no-duplicate-imports)